Responsibilities for GA (#3571)
Qiskit/documentation: docs/security/responsibilities.mdx

|Infrastructure health monitoring and notification| <ul><li>Remediate all infrastructure environment issues.</li><li> Notify customers of applicable incidents.</li></ul> | <ul><li> Ascertain the impact of each reported incident.</li><li>Engage [IBM Support](/docs/support) as required.</li></ul>|
|Maintain IBM Cloud high availability SLA| Operate Qiskit Runtime in accordance with the [IBM Cloud Public Service Level Agreements](https://cloud.ibm.com/docs/overview?topic=overview-slas) (SLAs). | Follow high availability best practices. |

## Change management

Change management includes tasks such as deployment, configuration, upgrades, patching, configuration changes, and deletion.

| Task | IBM responsibilities | Your responsibilities |
| --- | --- | --- |
| Major version upgrades | Provide availability and tools for major version updates. | Migrate to a supported version before end-of-support for an old version. |
| API version changes | Versioning the REST APIs. | Integrate with the wanted version of REST APIs. |

> | Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and take appropriate action.</li></ul> |

Reviewer: Should we link here: https://quantum.cloud.ibm.com/docs/guides/latest-updates ?

Author: I'll link to announcements instead since they appear directly on the dashboard.

> | Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and take appropriate action.</li></ul> |

Author: ```suggestion
| Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and [platform announcements,](https://quantum.cloud.ibm.com/announcements),and take appropriate action.</li></ul> |
```

> | Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and [platform announcements,](https://quantum.cloud.ibm.com/announcements),and take appropriate action.</li></ul> |

Author: ```suggestion
| Updates, fixes, and new features | <ul><li>Provide regular updates, bug fixes, and new features, transparently following a continuous delivery model.</li><li>Post notifications for changes that impact customers.</li></ul> | <ul><li>Set preferences to [receive email notifications](https://cloud.ibm.com/user/notifications) and monitor the [IBM Cloud status](https://cloud.ibm.com/status) page for general announcements.</li><li>Read system-generated notifications and [platform announcements,](https://quantum.cloud.ibm.com/announcements) and take appropriate action.</li></ul> |
```

## Identity and access management

Identity and access management includes tasks such as authentication, authorization, managing access control policies, and approving, granting, and revoking access.

| Task | IBM responsibilities | Your responsibilities |
| --- | --- | --- |

> | Identity and access control | Provide the ability to manage and restrict access to resources through IBM Cloud. | Grant, revoke, and manage access to resources through [Cloud Identity and Access Management](https://cloud.ibm.com/iam/overview) (IAM). |

Reviewer: Or we could link to https://quantum.cloud.ibm.com/docs/guides/access-groups

> | Identity and access control | Provide the ability to manage and restrict access to resources through IBM Cloud. | Grant, revoke, and manage access to resources through [Cloud Identity and Access Management](https://cloud.ibm.com/iam/overview) (IAM). |

Author: ```suggestion
| Identity and access control | Provide the ability to manage and restrict access to resources through IBM Cloud. | Grant, revoke, and manage access to resources through [Cloud Identity and Access Management](/docs/guides/access-groups) (IAM). |
```

| Vulnerabilities | Monitor to detect vulnerability and security compliance issues. | <ul><li>Keep informed about possible vulnerabilities and security issues through security bulletins that provide potential remediation actions.</li><li>Check the [IBM Cloud status website](https://cloud.ibm.com/status) to find announcements and security bulletins about key events that affect the IBM Cloud platform, infrastructure, and major services.</li></ul> |
| Encryption | Encrypt data when it is transmitted on any public networks and within the Cloud Service's private data center network. Encryption methods such as HTTPS and TLS are used to protect data in motion. | Ensure, as required, that your connection is encrypted end-to-end. |
| Security | <ul><li>Ensure the security of data on disk and data in motion within its infrastructure.</li><li>Provide a secure service that complies with key standards.</li></ul> | Restrict [user access](https://cloud.ibm.com/iam/overview) to the appropriate resources and resource groups. |

## Disaster recovery

| Task | IBM responsibilities | Your responsibilities |
| --- | --- | --- |

> | General | Restore or rebuild the provisioning environments and resources in the affected regions. | <ul><li>Monitor instance and resource states.</li><li>Provision new Qiskit Runtime service instances in an alternate region, if available.</li></ul> |

Reviewer: Could link here https://quantum.cloud.ibm.com/docs/en/guides/instances

> | General | Restore or rebuild the provisioning environments and resources in the affected regions. | <ul><li>Monitor instance and resource states.</li><li>Provision new Qiskit Runtime service instances in an alternate region, if available.</li></ul> |

Author: ```suggestion
| General | Restore or rebuild the provisioning environments and resources in the affected regions. | <ul><li>Monitor instance and resource states.</li><li>Provision new Qiskit Runtime service [instances](/docs/guides/instances) in an alternate region, if available.</li></ul> |
```
